File: Python/homework/homework001.py
import math

a = int(input())
b = int(input())
c = int(input())

d =  b*b-4*a*c

if (d < 0):
    print("無實根")
else:
    x1 = ((-b) + math.sqrt(b*b-4*a*c)) / (2*a)
    x2 = ((-b) - math.sqrt(b*b-4*a*c)) / (2*a)
    if x1 == x2:
        print('%.1f'%x1)
    else:
        print('%.1f'%x1)
        print('%.1f'%x2)

File: C/homework/homework001.c
#include <stdio.h>
#include <math.h>
#include <stdlib.h>

int main(void){
    int a,b,c;
    double x1,x2;
    int d;
    double A,B;
    scanf("%d",&a);
    scanf(" %d",&b);
    scanf(" %d",&c);

    d = (b*b)-4*a*c;

    if (d >= 0){
        x1 = ((-b)+sqrt(d))/(2*a);
        x2 = ((-b)-sqrt(d))/(2*a);
        printf("%.1f", x1);
        if (x1 != x2){
            printf("\n%.1f", x2);
        }
    }else{
        A = (-b)/(2*a);
        B = fabs(sqrt(abs(d))/(2*a));
        printf("%.1f+%.1fi", A,B);
        printf("\n%.1f-%.1fi", A,B);
    }
    return 0;
}